dmt/main/views.py

```diff
@@ -97,8 +97,11 @@ def get_context_data(self, **kwargs):
                 Q(comments__icontains=q)
             ),
             projects=Project.objects.filter(
-                Q(name__icontains=q) |
-                Q(description__icontains=q)
+                ~Q(status='Defunct') &
+                (
```

Comment:
My suggestion would be to use `,` between these statements rather than `&`.  In my mind `&` refers to [bitwise &](https://docs.python.org/3/reference/expressions.html#binary-bitwise-operations), but I think you mean logical and.

Looking at the docs, when you separate conditions with commas, it looks like Django implicitly adds `AND` to the underlying query: https://docs.djangoproject.com/en/1.11/ref/models/querysets/#django.db.models.query.QuerySet.filter

Reply:
We do generally use `&` for joining Q clauses in many places already, though, as this is documented as the way to do it - https://docs.djangoproject.com/en/1.11/ref/models/querysets/#q-objects so, i think this is fine. Yeah, it is kinda confusing that, in c-style syntax, `&&` is boolean AND, while `&` is bitwise.

Reply:
Well, hmm.. the docs also do this joined with a comma rather than an `&`. https://docs.djangoproject.com/en/1.11/topics/db/queries/#complex-lookups-with-q  Nevermind - AFAIK, either way is fine, i don't have a preference.

Reply:
Arg, using `&` makes the strongly-typed side of my brain scream (but I should lock him up back in his cage). Now that I understand the convention, either way is fine for me. @sdreher do you have any thoughts on the topic?